shared/build.gradle.kts

```diff
@@ -63,9 +63,13 @@ kotlin {
 			implementation(libs.workmanager)
 			implementation(libs.koin.android)
 			implementation(libs.koin.workmanager)
+			implementation(libs.ktor.engine.android)
```

Comment:
I had wrong feeling that it's not needed because images were loaded successfully, the reason behind is that images were placed in the cache so there was no attempt to download them 😄